src/java.security.jgss/share/classes/module-info.java

```diff
@@ -33,6 +33,7 @@
  */
 module java.security.jgss {
     requires java.naming;
+    requires java.security.sasl;
```

Comment:
Someone from security-dev should probably review this and validate that this is OK. I'm also a bit uncomfortable that we require a class from `com.sun.jndi.ldap.sasl` even though `java.naming` is already required by `java.security.jgss` - so maybe this is OK.

Reply:
Yes. I would like the security team to validate this.

Reply:
I suggest moving the `TlsChannelBinding` class into `java.base/sun.security.util` since it's not only used by LDAP anymore. You might need to modify the types of exceptions thrown in the class and move the 2 final strings to some other class inside `java.security.sasl`.

Reply:
Seems like `com.sun.jndi.ldap.sasl.TlsChannelBinding` is not misplaced....